lib/FluxibleContext.js

```diff
@@ -99,12 +99,27 @@ FluxContext.prototype.plug = function (plugin) {
 };
 
 /**
+ * Get action name, compatible with all browsers
+ * @param {Function} action
+ * @return {String} action display name
+ */
+function getActionName (action) {
+    var actionName = action.displayName || action.name;
+    if (!actionName) {
+        var match = action.toString().match(/^function\s*([^\s(]+)/);
```

Comment:
I thought `Function.name` was standard, but then I looked it up in [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name)... Stupid IE. (╯°□°）╯︵ ┻━┻